store/tikv/region_request.go

```diff
@@ -939,6 +947,7 @@ func (s *RegionRequestSender) onRegionError(bo *Backoffer, ctx *RPCContext, req
 		}
 		return retry, errors.Trace(err)
 	}
+
 	if regionErr.GetServerIsBusy() != nil {
 		logutil.BgLogger().Warn("tikv reports `ServerIsBusy` retry later",
```

Comment:
Should the log level be adjusted like others?

Reply:
I think it's qualified...